### ISSUE TITLE
Create new Release for Uli

### ISSUE THREAD

**Issue by dennyabrain** [tattle-made/Uli#394]
Lets release the crowdsourcing feature. Release should include 
- Tested feature on chrome and firefox
- new Release text on github with the changelog
- a blog post/article describing the release, which is made into a tweet too

Blockers : 
- Chrome auto release plugin is broken
- Backend server seems to be down
- Setting up staging server

**Comment by dennyabrain** [comment]
Aatman will create release this time. Come up with visuals and nice introductory text for the feature.

**Comment by aatmanvaidya** [comment]
@dennyabrain I figured out (hoefullly) why the github action is failing for chrome

I think our refresh token of the chrome store has expired, see this - https://github.com/Klemensas/chrome-extension-upload-action/issues/4
we are getting the same error as this

**Comment by dennyabrain** [comment]
oh you are right and thats known :)
Everytime before release I have to do these steps manually to get a new refresh token - https://github.com/fregante/chrome-webstore-upload/blob/main/How%20to%20generate%20Google%20API%20keys.md

So there's two related things : 
1. Is there a way to not have to do it manually everytime
2. I just tried to do this manually this time and looks like those steps dont work anymore :/

I found this guide also which goes over how the expiry of refresh tokens is an issue and he has a workaround  - https://cardinalby.github.io/blog/post/github-actions/webext/8-google-api-refresh-token-expiration/

But all this seems very roundabout to me. Its almost like google doesnt want us to automate this? Honestly i feel like keeping up with these might be more trouble than I care for. We already have automated the creation of the zip. and if its just a matter of uploading that zip to the store manually, that might be easier for me to do than worrying about all these changing refresh token nonsense.
